search.html

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
If `search.html` is the only place where we use the search form, would it make sense to just put the whole page in this file, and not have the include snippet at all?